### PR TITLE
Update Slack link

### LINKED ISSUE #395: Slack link no longer working
**Describe the bug or question**
I am unable to connect to slack via the link provided in overview (readme.md)

**To Reproduce**
Try to subscribe to the single-spa community in slack.
**Expected behavior**
Expect to be able to connect to slack community when clicking 'Join Slack Community'

### PR DESCRIPTION
**Pull request by filoxo**
Resolves #395

### CODE REVIEW

File: website/versioned_docs/version-4.x/contributing-overview.md [hunk 1]
```diff
@@ -11,40 +11,40 @@ We've put together the following guidelines to help you figure out where you can
...
 
 0. [Types of contributions we're looking for](#types-of-contributions-were-looking-for)
-0. [Ground rules & expectations](#ground-rules-expectations)
-0. [How to contribute](#how-to-contribute)
-0. [Setting up your environment](#setting-up-your-environment)
-0. [Community](#community)
+1. [Ground rules & expectations](#ground-rules-expectations)
+2. [How to contribute](#how-to-contribute)
+3. [Setting up your environment](#setting-up-your-environment)
+4. [Community](#community)
```
Comment (frehner) [inline website/versioned_docs/version-4.x/contributing-overview.md]: it's sometimes nice to have the list with `0`s - if you ever need to rearrange/delete/insert a new item in any location, you don't have to renumber the whole list to make that update; you can just make the change and markdown will automatically number them correctly.

not a big deal though.

**Review by frehner — APPROVED** [review]
non-blocking comment

File: website/versioned_docs/version-4.x/contributing-overview.md (hunk 1 above)
Comment (filoxo) [reply website/versioned_docs/version-4.x/contributing-overview.md]: Oh yea, interesting, that was done by prettier. I'll see if I can revert it.

Comment (filoxo) [reply website/versioned_docs/version-4.x/contributing-overview.md]: Oh interesting. If you begin the list with `0` prettier will autoformat it as a sequence. If you start with `1` and keep going with just `1`s, they'll be preserved as-is while still keeping the old behavior.

Comment (frehner) [reply website/versioned_docs/version-4.x/contributing-overview.md]: ah good to know - I think I generally use `1`s to do this as well, so we can just stick with `1`s

**Pull request merged**